### PR TITLE
prepare 0.54

### PR DESCRIPTION
**Pull request by fpetkovski**
## Description

Prepares the 0.54 release. 

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Comment by slashpai** [comment]
We can include https://github.com/prometheus-operator/prometheus-operator/pull/4491also in this release?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 0.54.0 / 2021-01-26
+
+* [FEATURE] Support SNS Receiver in AlertmanagerConfig CR. #4468
+* [ENHANCEMENT] Specify SA token automounting on pod-level for operator and prometheus operand. #4514
+* [ENHANCEMENT] Support following redirects and Oauth2 in HTTP Client config in raw alertmanager config secret. #4499
+* [ENHANCEMENT] Add validations for relabel configs. #4429
```
Comment (simonpasquier) [inline CHANGELOG.md:6]: that could be considered a BUGFIX

Comment (fpetkovski) [reply CHANGELOG.md]: Makes sense, it would be consistent with the other validation changes.

**Comment by fpetkovski** [comment]
> We can include #4491 also in this release?

Sure thing, added :+1:

**Review by PhilipGough — APPROVED** [review]
lgtm

**Review by simonpasquier — APPROVED** [review]
lgtm

**Review by paulfantom — APPROVED** [review]
:shipit:

**Pull request merged**